### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by iamgaborgithub**
And I'm interested in to update the "DOCUMENT FORMAT DESCRIPTION COMES LATER. Until then, we are providing some example JSONs in the jsons folder in this repository." part because it isn't true until we don't have a folder called "jsons".

And I prefer to have three small paragraphs about the "elements", "full-email-layout-examples" and "property-groups" folders to introduce why they are there.

### CODE REVIEW

**Review by blepati — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ With the settings object you can change the behaviour of the generator. Hopefull
 		vmlBackground: true,
 		rolePresentation: false,
 		lineLength: 799,
+		encodeUrls: true,
+		"lang": "fr"
```
Comment (gyulanemeth) [inline README.md:51]: We did not use "-s at the prop names in the example. Plz don't mix styles :)

File: compatibility/README.md [hunk 2]
```diff
@@ -59,6 +59,6 @@ Windows Phone 7.5
 
 
 Notes:
-*Outlook 2007-2019 only renders background images if it is coded in VML, there is an option in the generator to made both VML and HTML declaration for the background. [Read more](../generator-settings/VMLbackground.md)
+*Outlook 2007-2019 only renders background images if it is coded in VML, there is an option in the generator to made both VML and HTML declaration for the background. [Read more](../document-settings/VMLbackground.md)
 
- If you have any questions or run into any issues you can create an issue [here](https://github.com/EDMdesigner/editor-issues) 
+ If you have any questions or run into any issues you can create an issue [here](https://github.com/EDMdesigner/editor-issues/issues)
```
Comment (gyulanemeth) [inline compatibility/README.md]: I just realized that the is leading to another repository. Shall we not handle generator issues in this repository?

**Review by gyulanemeth — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (iamgaborgithub) [reply README.md:51]: If you sure that it works without "s" then I'll change that property everywhere.
example: https://github.com/EDMdesigner/email-generator-docs/blob/master/document-settings/urlEncoding.md

File: compatibility/README.md (hunk 2 above)
Comment (iamgaborgithub) [reply compatibility/README.md]: I agree I'll check this repository's settings about that and change every URL to point here.

File: README.md (hunk 1 above)
Comment (blepati) [reply README.md:51]: my bad, I'll fix these and make it consistent

**Pull request merged**